app/views/course/forum/posts/_post.html.slim

```diff
@@ -19,9 +19,10 @@
       - if defined?(show_buttons)
         div.pull-right
           .btn-group
-            = link_to reply_course_forum_topic_post_path(current_course, @forum, @topic, post),
-                      title: t('.reply'), class: ['btn', 'btn-primary'] do
-              = fa_icon 'mail-reply'.freeze
+            - if post.parent_id == nil
```

Comment:
Add a comment to explain the reason for this, because at first glance it's strange to hide a button based on `parent.id.nil?`